docs/examples/userguide/language_basics/casting_python.py

```diff
@@ -12,7 +12,7 @@ def main():
     adress_in_c = cython.cast(Py_intptr_t, ptr)
     address_from_void = adress_in_c        # address_from_void is a python int
 
-    ptr2 = cython.cast(cython.pointer(PyObject), python_string)
+    ptr2 = cython.cast(cython.pointer[PyObject], python_string)
```

Comment:
This seems something that we should support more simply as `p_PyObject`. Doesn't have to be in this PR.